roles/edpm_neutron_metadata/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# defaults file for edpm_ovn
+
+# seconds between retries for download tasks
+edpm_neutron_metadata_images_download_delay: 5
```

Comment:
would be good to have the variable names consistent with others in the file, something like:-
edpm_ovn_metadata_images_download_delay
edpm_ovn_metadata_images_download_retries

may be also rename the role to edpm_ovn_metadata  or rename the vars to edpm_neutron_metadata*?

Reply:
I would go for edpm_neutron_metadata, as we have there "edpm_neutron_sriov" and "edpm_neutron_dhcp" already.  So, I'll go for renaming the vars

Reply:
also, this will require changes to install_yamls

Reply:
sorry, not install_yamls... defaults values are coming from dataplane-operator... so I'll make the changes there then